projects/packages/my-jetpack/_inc/components/product-detail-card/index.jsx

```diff
@@ -156,91 +122,75 @@ const ProductDetailCard = ( { slug, onClick, trackButtonClick, className } ) =>
-						{ title }
-					</Text>
+				<H3>{ title }</H3>
+				<Text mb={ 3 }>{ longDescription }</Text>
 
-					<Text className={ styles.name } component={ Col }>
-						{ longDescription }
-					</Text>
-
-					<Col>
-						<ul className={ styles.features }>
-							{ features.map( ( feature, id ) => (
-								<Text component="li" key={ `feature-${ id }` } variant="body-extra-small">
-									<Icon icon={ check } size={ 30 } />
-									{ feature }
-								</Text>
-							) ) }
-						</ul>
-					</Col>
-
-					{ needsPurchase && (
-						<Col className={ styles[ 'price-container' ] }>
-							<Price value={ price } currency={ currencyCode } isOld={ true } />
-							<Price value={ discountPrice } currency={ currencyCode } isOld={ false } />
-							<Text className={ styles[ 'price-description' ] }>
-								{ __( '/month, paid yearly', 'jetpack-my-jetpack' ) }
-							</Text>
-						</Col>
-					) }
-
-					{ isFree && (
-						<Text variant="title-small" component={ Col }>
-							{ __( 'Free', 'jetpack-my-jetpack' ) }
+				<ul className={ styles.features }>
+					{ features.map( ( feature, id ) => (
+						<Text component="li" key={ `feature-${ id }` } variant="body">
+							<Icon icon={ check } size={ 30 } />
+							{ feature }
 						</Text>
-					) }
-
-					<Col>
-						{ ( ! isBundle || ( isBundle && ! hasRequiredPlan ) ) && (
-							<Text
-								component={ ProductDetailButton }
-								onClick={ clickHandler }
-								isLoading={ isFetching }
-								isPrimary={ ! isBundle }
-								href={ onClick ? undefined : addProductUrl }
-								className={ styles[ 'checkout-button' ] }
-								variant="body"
-							>
-								{
-									/* translators: placeholder is product name. */
-									sprintf( __( 'Add %s', 'jetpack-my-jetpack' ), title )
-								}
-							</Text>
-						) }
-
-						{ isBundle && hasRequiredPlan && (
-							<div className={ styles[ 'product-has-required-plan' ] }>
-								<CheckmarkIcon size={ 36 } />
-								<Text>{ __( 'Active on your site', 'jetpack-my-jetpack' ) }</Text>
-							</div>
-						) }
-					</Col>
-				</Container>
-			</Col>
-		</Container>
+					) ) }
+				</ul>
+
+				{ needsPurchase && (
+					<div className={ styles[ 'price-container' ] }>
+						<Price value={ price } currency={ currencyCode } isOld={ true } />
+						<Price value={ discountPrice } currency={ currencyCode } isOld={ false } />
+						<Text className={ styles[ 'price-description' ] }>
+							{ __( '/month, paid yearly', 'jetpack-my-jetpack' ) }
+						</Text>
+					</div>
+				) }
+
+				{ isFree && <H3>{ __( 'Free', 'jetpack-my-jetpack' ) }</H3> }
+
+				{ ( ! isBundle || ( isBundle && ! hasRequiredPlan ) ) && (
+					<Text
+						component={ ProductDetailButton }
+						onClick={ clickHandler }
+						isLoading={ isFetching }
+						isPrimary={ ! isBundle }
+						href={ onClick ? undefined : addProductUrl }
+						className={ styles[ 'checkout-button' ] }
+						variant="body"
```

Comment:
I left this when refactored it, but I think we could remove it since `body` is the default one.

Reply:
agree